bootstrap/pom.xml

```diff
@@ -21,7 +21,7 @@
     <dependency>
       <groupId>org.jenkins-ci</groupId>
       <artifactId>version-number</artifactId>
-      <version>1.7</version>
+      <version>1.6</version>
```

Comment:
this should come from the jenkins bom

Reply:
Yes. This is what I am checking right now

Reply:
@timja BOM does not include it at the moment. https://github.com/jenkinsci/jenkins/blob/master/bom/pom.xml . Maybe a quick fix